src/containers/header/index.js

```diff
@@ -29,15 +27,15 @@ const Header = props => {
   }, []);
 
   useEffect(() => {
-    if (width > 768) {
+    if (width > 992) {
       const toggleElement = document.querySelector('#menu');
       toggleElement.classList.remove('mobileMenuActive');
       setMenuStatus(false);
     }
   });
 
   const toggleMenuFunc = () => {
-    if (width <= 768) {
+    if (width <= 992) {
```

Comment:
Is it necessary to use JS for implement Adaptive for this component?

Reply:
Dunno, this part isn't connected with my task.